Presentation.Web/Presentation.Web.csproj

```diff
@@ -69,8 +69,9 @@
     <Reference Include="Castle.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=407dd0808d44fbdc, processorArchitecture=MSIL">
       <HintPath>..\packages\Castle.Core.4.4.1\lib\net45\Castle.Core.dll</HintPath>
     </Reference>
-    <Reference Include="dk.nita.saml20, Version=2.0.5.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\dk.nita.saml20.2.0.5\lib\net472\dk.nita.saml20.dll</HintPath>
+    <Reference Include="dk.nita.saml20, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\OIOSAML.Net\src\dk.nita.saml20\dk.nita.saml20\bin\Debug\dk.nita.saml20.dll</HintPath>
```

Comment:
Perhaps add the TECH story on distributing our own version of the OIOSAML.Net package on nuget